File: TextosJapones/src/main/java/org/jisho/textosJapones/model/enums/Dicionario.java
package org.jisho.textosJapones.model.enums;

public enum Dicionario {
	SAMLL("Sudachi - small"), CORE("Sudachi - core"), FULL("Sudachi - full");

	private final String dicionario;

	Dicionario(String dicionario) {
		this.dicionario = dicionario;
	}

	public String getDescricao() {
		return dicionario;
	}

	@Override
	public String toString() {
		return this.dicionario;
	}
}

File: ProcessaTexto/src/main/kotlin/br/com/fenix/processatexto/model/enums/Dicionario.kt
package br.com.fenix.processatexto.model.enums

enum class Dicionario(val descricao: String) {
    SAMLL("Sudachi - small"), CORE("Sudachi - core"), FULL("Sudachi - full");

    @Override
    override fun toString(): String {
        return descricao
    }
}